helm-charts/CI.md

```diff
@@ -0,0 +1,26 @@
+# CI guidelines for helm charts
+
+## Table of Contents
+
+- [Infra Setup](#infra-setup)
+- [Add new test case](#add-new-test-case)
+
+## Infra Setup
+
+Helm charts are orginized as:
+1). E2E examples are directly located under helm-charts/ directory
+2). Charts for components are located under helm-charts/common directory.
+
+We have Xeon servers with Gaudi accelerator card, so basicly we'll cover running workloads for cpu only and with Gaudi accelerated.For other configurations/feature coverage, it's case by case.
```

Comment:
There's typo, but IMHO this can be simplified to:
```suggestion
We have Xeon servers with Gaudi accelerator card, which covers CPU and Gaudi accelerated workloads. For coverage of other configurations/features, it's case by case.
```